src/test/java/racingcar/WinnersTest.java

```diff
@@ -0,0 +1,40 @@
+package racingcar;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class WinnersTest {
+
+    @Test
+    @DisplayName("[성공] 자동차 경주의 최종 우승자가 한명인 경우 한명의 이름만 반환한다.")
+    void 자동차_경주_우승자() {
+        // Given
+        Cars cars = new Cars(
+                new Car(new Distance(3), new Name("pobi")),
```

Comment:
`Car` 인스턴스를 조금 더 쉽게 생성할 수 있도록 생성자를 통해 편의성을 제공해보면 어떨까요?
```suggestion
                new Car(3, "pobi"),
```